shell/app-shell/elements/arc-planner.js

```diff
@@ -31,7 +31,7 @@ class ArcPlanner extends Xen.Debug(Xen.Base, log) {
       invalid: 0
     };
   }
-  _willReceiveProps(props, state, oldProps) {
+  async _willReceiveProps(props, state, oldProps) {
```

Comment:
looks like there's an optional 4th param lastState
also appears that this is never await'd in xen-state.js?

I don't know this code very well, so I could be wrong.

Reply:
I think the 4th parameter is changed in other places. Anyway, it is mostly irrelevant for this change, imo

Reply:
ack